client/src/pages/Login.vue

```diff
@@ -5,15 +5,12 @@
         <div class="text-h5 text-white">Login</div>
       </q-card-section>
       <q-card-section class="q-pa-lg">
-        <q-form
-          v-on:submit.prevent="onSubmit()"
-          class="q-px-sm q-pt-md q-pb-lg"
-        >
+        <q-form v-on:submit.prevent="login()" class="q-px-sm q-pt-md q-pb-lg">
           <q-input
             square
-            ref="username"
-            v-model="form.username"
-            :label="$t('auth.login_fields.username')"
+            ref="email"
+            v-model="form.email"
+            :label="$t('auth.login_fields.email')"
             @keypress.enter="$refs.password.focus()"
```

Comment:
Add autocomplete="username"

https://web.dev/sign-in-form-best-practices/

Reply:
I've added this missing autocomplete attribute to the email input. It's wild that "username" is necessary for email inputs according to the documentation you linked!